src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -1413,6 +1420,8 @@ internal partial class BoundDynamicIndexerAccess
     {
         protected override OperationKind ExpressionKind => OperationKind.None;
 
+        protected override ImmutableArray<IOperation> Children => this.Arguments.Add(this.ReceiverOpt).As<IOperation>();
```

Comment:
>this.ReceiverOpt [](start = 85, length = 16)

Receiver should probably be the first. We also probably don't want to add nulls to the list.